### ISSUE TITLE
Create a jinja connivence function for the defined test

### ISSUE THREAD

**Issue by ryanmerolle** [aristanetworks/ansible-avd#3324]
### Enhancement summary

publish as a separate package

### Which component of AVD is impacted

plugins

### Use case example

This is a super useful plugin that could easily get published as its own filter for python, ansible, etc.

### Describe the solution you would like

publish as a separate package

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

**Comment by ryanmerolle** [comment]
you could also just move it to pyavd.

**Comment by ClausHolbechArista** [comment]
I don't think we will move on this anytime soon. For now you can import it from `pyavd.vendor.j2.test.defined` but notice this may change as we move code to pyavd instead of vendoring it in.
